nvmw.bat

```diff
@@ -75,27 +81,44 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node

```

Comment:
OK, but this is breaking change. So we should add document about this change and how to migration from old nvmw.